### ISSUE TITLE
Clarification on dimensions of confusion matrix (binary y variable)

### ISSUE THREAD

**Issue by carlocelis** [InseadDataAnalytics/INSEADAnalytics#136]
Something that may be useful for the final project...

In the example used in the slide below, does the dependent variable equal 1 if the customer is retained or not? I think this is important because the relevant metric, either sensitivity or specificity, can be interchanged depending on how the dependent variable is defined. 

![image](https://user-images.githubusercontent.com/39317651/40364792-7278fcde-5dd3-11e8-8d83-4f8966133d35.png)

Also, does R's output always follow the following format?

![image](https://user-images.githubusercontent.com/39317651/40365342-a2014b36-5dd4-11e8-9c91-86829def2af7.png)

**Comment by Antoine-Engerand** [comment]
Yes both points are correct, to my understanding. R recognizes 1s & 0s and so you may not really care about in which format R outputs confusion matrices.

**Comment by carlocelis** [comment]
Thanks! I think the format is important because if I decide to swap the definition of the y variable (e.g. in the Credit card case, assign 0 to mean customer defaults next month instead of 1), then I will be more concerned about sensitivity instead of specificity
